ginsu-h2/src/main/java/com/salesforce/ginsu/persist/h2/H2PersistenceProviderBuilder.java

```diff
@@ -20,7 +20,7 @@
 public class H2PersistenceProviderBuilder {
 
     private static final String DEFAULT_JDBC_DRIVER = "org.h2.Driver";
-    private static final String DEFAULT_CONNECTION_STRING_FORMAT = "jdbc:h2:%s;CACHE_SIZE=131072";
+    private static final String DEFAULT_CONNECTION_STRING_FORMAT = "jdbc:h2:%s;CACHE_SIZE=131072;AUTO_SERVER=TRUE";
```

Comment:
The latest version of H2 enforces single connection in embedded mode. Multiple connections (as required by multiple threads) require TCP connection